test/functional/test_framework/wallet.py

```diff
@@ -50,6 +51,7 @@
     key_to_p2wpkh_script,
     keyhash_to_p2pkh_script,
     scripthash_to_p2sh_script,
+    program_to_witness_script
```

Comment:
styling nit: should be lexicographically sorted and add a comma at the end of each line (even if it is the last one)

Reply:
Thanks. :+1: